x-pack/plugin/ent-search/qa/rest/src/yamlRestTest/resources/rest-api-spec/test/entsearch/55_search_application_search.yml

```diff
@@ -113,13 +122,11 @@ teardown:
 "Query Search Application with default parameters":
   - skip:
       features: headers
-      version: all
-      reason: "AwaitsFix https://github.com/elastic/enterprise-search-team/issues/4540"
 
   - do:
       headers: { Authorization: "Basic ZW50c2VhcmNoLXVzZXI6ZW50c2VhcmNoLXVzZXItcGFzc3dvcmQ=" }  # user
       search_application.search:
-        name: test-search-application
+        name: test-search-application-1
```

Comment:
The initial test was failing because we marked `field_name` as `required` in `test-search-application`. `another-test-search-application` was never used in this test, so I replaced it with a new test search application that omitted the `required` parameter.